corehq/form_processor/utils.py

```diff
@@ -33,3 +35,12 @@ def _inner(*args, **kwargs):
         return _wrap(obj)
 
     return _inner
+
+
+def should_use_sql_backend(domain):
+    toggle_to_check = USE_SQL_BACKEND
```

Comment:
I meant the variable part of it specifically, given that it's only used once, and there's no forking on what gets assigned to `toggle_to_check`.